Promtail doc (#563)
grafana/agent: docs/windows.md

## Overview

> The installer will install Grafana Agent into the default directory `C:\Program Files (x86)\Grafana Agent`. The [windows_exporter](https://github.com/grafana/windows_exporter) can be enabled with all default windows_exporter options. 

Reviewer: ```suggestion
The installer will install Grafana Agent into the default directory `C:\Program Files (x86)\Grafana Agent`. The [windows_exporter integration](https://github.com/prometheus-community/windows_exporter) can be enabled with all default windows_exporter options. 
```

Let's link to the official project here IMO

## Installation

After installation, ensure that you can reach `http://localhost:12345/-/healthy` and `http://localhost:12345/agent/api/v1/targets`. 

> If Grafana Agent is re-installed and an agent-config.yaml already exists; the installation will not overwrite the existing one.

Reviewer: ```suggestion
Existing configuration files will be kept when re-installing or upgrading the Grafana Agent.
```

Reviewer: (And then move this paragraph below the next one, I think)

## Logs

> When Grafana Agent runs as a Windows Service, the Grafana Agent will write logs to Windows Event Logs. When running as executable, Grafana Agent will write to standard out. The logs will be written with the source of `Grafana Agent`.

Reviewer: Can you add the eventlog name we use here? That'd make it easier for people to set up the Agent to collect logs from its own service.

> ## Pushing Windows Logs to Loki

Reviewer: ```suggestion
## Pushing Windows logs to Loki
```
